aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -448,3 +467,48 @@ resource "aws_elasticsearch_domain" "example" {
 }
 `, randInt)
 }
+
+func testAccESDomainConfig_vpc(randInt int) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_default_vpc" "default" {}
+
+resource "aws_default_subnet" "first" {
+  availability_zone = "${data.aws_availability_zones.available.names[0]}"
+}
+
+resource "aws_default_subnet" "second" {
+  availability_zone = "${data.aws_availability_zones.available.names[1]}"
+}
+
+resource "aws_security_group" "first" {
+  vpc_id = "${aws_default_vpc.default.id}"
+}
```

Comment:
To avoid sharing any resources and allow running multiple tests in parallel do you mind building custom VPC & subnets here, instead of creating default ones?